packages-private/vapor-e2e-test/__tests__/vdomInterop.spec.ts

```diff
@@ -1,146 +1,115 @@
 
-      expect(await text('.vdom-prop')).toContain('hello')
+    expect(css('.vdom-prop')).toHaveTextContent('hello')
 
-      const tt = await text('.vapor-slot-in-vdom-default')
-      expect(tt).toContain('slot prop: slot prop')
-      expect(tt).toContain('component prop: hello')
+    const tt = css('.vapor-slot-in-vdom-default')
+    expect(tt).toHaveTextContent('slot prop: slot prop')
+    expect(tt).toHaveTextContent('component prop: hello')
 
-      await click('.change-vapor-slot-in-vdom-prop')
-      expect(await text('.vapor-slot-in-vdom-default')).toContain(
-        'slot prop: changed',
-      )
+    await css('.change-vapor-slot-in-vdom-prop').click()
+    expect(css('.vapor-slot-in-vdom-default')).toHaveTextContent(
+      'slot prop: changed',
+    )
 
-      expect(await text('.vapor-slot-in-vdom-test')).toContain('fallback')
+    expect(css('.vapor-slot-in-vdom-test')).toHaveTextContent('fallback')
 
-      await click('.toggle-vapor-slot-in-vdom-default')
-      expect(await text('.vapor-slot-in-vdom-default')).toContain(
-        'default slot fallback',
-      )
+    await css('.toggle-vapor-slot-in-vdom-default').click()
+    expect(css('.vapor-slot-in-vdom-default')).toHaveTextContent(
+      'default slot fallback',
+    )
 
-      await click('.toggle-vapor-slot-in-vdom-default')
+    await css('.toggle-vapor-slot-in-vdom-default').click()
 
-      await enterValue('input', 'bye')
-      expect(await text('.vapor-prop')).toContain('bye')
-      expect(await text('.vdom-slot-in-vapor-default')).toContain('bye')
-      expect(await text('.vdom-prop')).toContain('bye')
-      expect(await text('.vapor-slot-in-vdom-default')).toContain('bye')
-    },
-    E2E_TIMEOUT,
-  )
+    await css('input').fill('bye')
+    expect(css('.vapor-prop')).toHaveTextContent('bye')
+    expect(css('.vdom-slot-in-vapor-default')).toHaveTextContent('bye')
+    expect(css('.vdom-prop')).toHaveTextContent('bye')
+    expect(css('.vapor-slot-in-vdom-default')).toHaveTextContent('bye')
+  })
 
   describe('vdom transition', () => {
-    test(
-      'render vapor component',
-      async () => {
-        const btnSelector = '.trans-vapor > button'
-        const containerSelector = '.trans-vapor > div'
+    test('render vapor component', { timeout: E2E_TIMEOUT }, async () => {
+      const btnSelector = '.trans-vapor > button'
+      const containerSelector = '.trans-vapor > div'
 
-        expect(await html(containerSelector)).toBe(`<div>vapor compA</div>`)
+      expect(css(containerSelector).element().innerHTML).toBe(
+        `<div>vapor compA</div>`,
+      )
 
-        // comp leave
-        expect(
-          (await transitionStart(btnSelector, containerSelector)).innerHTML,
-        ).toBe(
-          `<div class="v-leave-from v-leave-active">vapor compA</div><!---->`,
-        )
+      // comp leave
+      await css(btnSelector).click()
+      expect(css(containerSelector).element().innerHTML).toBe(
```

Comment:
Accessing `element()` is also not a good idea, checking `toContainHTML` is more resilient